modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/VersionableContentDashboardItem.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.dashboard.web.internal.item;
+
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+import java.util.List;
+
+/**
+ * @author Cristina González
+ */
+public interface VersionableContentDashboardItem<T>
+	extends ContentDashboardItem<T> {
+
+	public List<ContentDashboardItem.Version> getAllVersions(
```

Comment:
Does that method make sense anymore?

Reply:
Hi @cgoncas, no, it's the method that we'll remove

Reply:
Totally true!!!